rllib/env/single_agent_env_runner.py

```diff
@@ -818,7 +818,7 @@ def _log_episode_metrics(self, length, ret, sec):
         )
 
         # For some metrics, log min/max as well.
-        self.metrics.log_value(EPISODE_LEN_MIN, length, reduce="min")
-        self.metrics.log_value(EPISODE_RETURN_MIN, ret, reduce="min")
-        self.metrics.log_value(EPISODE_LEN_MAX, length, reduce="max")
-        self.metrics.log_value(EPISODE_RETURN_MAX, ret, reduce="max")
+        self.metrics.log_value(EPISODE_LEN_MIN, length, reduce="min", window=win)
```

Comment:
Awesome! That's why I always felt that min/max stats were lifetime (rather than win=100) :D 
Makes perfect sense. Thanks for this fix @simonsays1980 !